### PR TITLE
validation for files upload and multiple file uploads enabled

### PR DESCRIPTION
**Pull request by naveen17797**
1. validation for image files
2. validation fix for pdf files
3. multiple file upload option enabled

### CODE REVIEW

**Comment by aethelwulffe** [comment]
Naveen, this feature might be a dead one.
Would you please check the code in /modules/filemanager (almost identical to sites/default/filemanager) and see what you think about improving these as a general file manager in the system?  Currently, they are two blended and sandboxed tools built out of sample code.  The one under /sites exists to enable the ability to upload .sql files, and retrieve .xml report exports from the feature in /modules/MIPS.  That feature is enabled by the "demo" flag in Globals/MIPS.  The file manager copy in /modules/filemanager is not pointed at by the .json menu by default.  The hope has been that a tool like this, perhaps using the same .js library could replace all the file upload/download features for billing files, patient documents, as well as exported report copies.

**Comment by naveen17797** [comment]
i had my opensource project called newway, https://github.com/naveen17797/newway-filemanage, it is written in php and jquery.
will it be useful?
below is the screenshot of my open source project
screenshot from 2018-02-28 20-55-15
screenshot from 2018-02-28 20-54-58

File: interface/super/manage_site_files.php [hunk 1]
```diff
@@ -211,5 +228,4 @@ function msfFileChanged() {
 
 </form>
 </body>
-</html>
-
+</html>
```
Comment (aethelwulffe) [inline interface/super/manage_site_files.php]: Newline at end of file please.

**Comment by aethelwulffe** [comment]
@naveen17797 You wanna look at using your project to replace the filemanger in /modules and in the sites directory?  Currently used with /modules/MIPS?

**Pull request merged**